README.rst

```diff
@@ -4,6 +4,8 @@ pex_config
 
 .. image:: https://img.shields.io/pypi/v/lsst-pex-config.svg
     :target: https://pypi.org/project/lsst-pex-config/
+.. image:: https://github.com/lsst/pex_config/actions/workflows/build.yaml/badge.svg
```

Comment:
I don't understand what this gives me? This should always be reporting test passing because we aren't allowed to merge to main without it. Was this showing failed on main because of the PyPI upload failure for any time where main and a tag coincided?

Reply:
No, the last step doesn't get run when you merge to main, but only on tag. The status of this badge would have turned to failed (in red). Similar to how the red X shows up next to the most recent commit on `main`.

This is the workflow log that it would report: https://github.com/lsst/pex_config/actions/runs/5276812239/jobs/9544053729

Reply:
I'm still confused because that link at the moment shows "build_and_test: passing" -- it does not show the failure that I can see if I go to the main GitHub page and click on the little red "x" against current main commit.

Reply:
It shows green now because the most recent run of this workflow passed (and didn't try uploading to PyPI). If I merged just the badge tonight and hold off the main fix until tomorrow, I believe it would turn red by then.

Reply:
If it always is showing the most recent build from a PR unrelated to main I don't think we want this. It doesn't tell people anything actionable.

Reply:
Fair enough, removed.